src/auth/src/lib.rs

```diff
@@ -15,7 +15,7 @@
 mod common;
 pub mod error;
 mod permission;
-mod user_info;
+pub mod user_info;
```

Comment:
I think we should be very careful to open this crate up. We should only be using the public `UserInfo` trait and not explicitly use any implementation struct in codebase.